examples/kubernetes/1.12/cilium-crio-ds.yaml

```diff
@@ -0,0 +1,189 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cilium
+  namespace: kube-system
+spec:
+  updateStrategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      # Specifies the maximum number of Pods that can be unavailable during the update process.
+      # The current default value is 1 or 100% for daemonsets; Adding an explicit value here
+      # to avoid confusion, as the default value is specific to the type (daemonset/deployment).
+      maxUnavailable: "100%"
+  selector:
+    matchLabels:
+      k8s-app: cilium
+      kubernetes.io/cluster-service: "true"
+  template:
+    metadata:
+      labels:
+        k8s-app: cilium
+        kubernetes.io/cluster-service: "true"
+      annotations:
+        # This annotation plus the CriticalAddonsOnly toleration makes
+        # cilium to be a critical pod in the cluster, which ensures cilium
+        # gets priority scheduling.
+        # https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/tolerations: >-
+          [{"key":"dedicated","operator":"Equal","value":"master","effect":"NoSchedule"}]
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9090"
+    spec:
+      serviceAccountName: cilium
+      initContainers:
+      - name: clean-cilium-state
+        image: docker.io/library/busybox:1.28.4
+        imagePullPolicy: IfNotPresent
+        command: ['sh', '-c', 'if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; rm -rf /sys/fs/bpf/tc/globals/cilium_*; fi']
+        volumeMounts:
+          - name: bpf-maps
+            mountPath: /sys/fs/bpf
+          - name: cilium-run
+            mountPath: /var/run/cilium
+        env:
+          - name: "CLEAN_CILIUM_STATE"
+            valueFrom:
+              configMapKeyRef:
+                name: cilium-config
+                optional: true
+                key: clean-cilium-state
+      containers:
+      - image: docker.io/cilium/cilium:latest
```

Comment:
do
```
git rebase -i 1735eebf07048c87677dce3696aa59cc7a28369f^
```
set it edit this commit
then do `cd examples/kubernetes && make clean && make CILIUM_VERSION=1.1.0 && git commit --amend && git rebase --continue`

Reply:
Andre, when you say edit the commit i.e set it to option `e`
which commit are you referring to?
This is what I get
```
pick 1735eebf0 examples/kubernetes: add k8s 1.12 deployment files
pick 3a9379f3f test: set default CRI socket
pick ad9066d45 ginkgo.Jenkinsfile: increase timeout by 30 minutes
pick b222f596e pkg/endpoint: set policy revision if there is no datapath changes
pick b52831b5d tests: Update cilium-builder in unit tests to 2018-06-21
pick 1c1cb3d81 test: Use latest stable etcd and consul images
pick 6f9fe52a8 backport: use the same url for all searches
pick adfe34fdf backport: Only check merged PRs
pick 4fe7d8f87 pkg/policy: take into account To / FromRequires when computing L4 policy
```

Reply:
Nevermind...

I am getting the following error:
```
vagrant@runtime1:~/go/src/github.com/cilium/cilium$ cd examples/kubernetes && make clean && make CILIUM_VERSION=1.1.0 && git commit --amend && git rebase --continue
for k8s_version in 1.7 1.8 1.9 1.10 1.11 1.12; do  rm -f ./$k8s_version/*.yaml; done
"1.1.0" is not a valid version. Must match regexp '(v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-.+)?)|(latest)'. Please fix VERSION file
Makefile:53: recipe for target '1.7/cilium-cm.yaml' failed
make: *** [1.7/cilium-cm.yaml] Error 1
vagrant@runtime1:~/go/src/github.com/cilium/cilium/examples/kubernetes$ 
```

Reply:
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Reply:
Dropping PR: #4265 -- test: add k8s 1.12 test framework (@aanm) as per offline discussion with @aanm